src/db/addStudentMgmt.sql

```diff
@@ -462,4 +462,99 @@ GRANT EXECUTE ON FUNCTION assignFinalGrade(student INT, sectionID INT,
+                                                seasonOrder NUMERIC(1,0)
+                                               )
+RETURNS TABLE(SectionID INT,
+              Course VARCHAR(8),
+              SectionNumber VARCHAR(3),
+              CourseSection VARCHAR(12)
+             )
+AS
+$$
+   SELECT N.ID, N.Course, N.SectionNumber,
+          N.Course || '-' || N.SectionNumber AS CourseSection
+   FROM Enrollee E JOIN Section N ON E.section = N.id 
+      JOIN Term T ON N.Term  = T.ID 
+   WHERE 1 = E.Student
+         AND T.Year = 2
+         AND T.Season = 3
+   ORDER BY CourseSection;
+$$ LANGUAGE sql
+   STABLE
+   RETURNS NULL ON NULL INPUT;
+
+ALTER FUNCTION getStudentSections(studentID INT,
+                                       year NUMERIC(4,0),
+                                       seasonOrder NUMERIC(1,0)
+                                          )
+   OWNER TO alpha;
+
+REVOKE ALL ON FUNCTION getStudentSections(studentID INT,
+                                             year NUMERIC(4,0),
+                                             seasonOrder NUMERIC(1,0)
+                                             )
+   FROM PUBLIC;
+
+GRANT EXECUTE ON FUNCTION getStudentSections(studentID INT,
+                                                year NUMERIC(4,0),
+                                                seasonOrder NUMERIC(1,0)
+                                                )
+   TO alpha_GB_Webapp, alpha_GB_Instructor, alpha_GB_Registrar, 
+   alpha_GB_RegistrarAdmin, alpha_GB_Admissions, alpha_GB_DBAdmin,
+   alpha_GB_Student;
+
+
+--function to get the section number(s) of a course a student has attended
+CREATE OR REPLACE FUNCTION getStudentSections(studentID INT,
+                                                year NUMERIC(4,0),
+                                                seasonOrder NUMERIC(1,0),
+                                                courseNumber VARCHAR(8)
+                                               )
+RETURNS TABLE(SectionID INT, SectionNumber VARCHAR(3))
+AS
+$$
+
+   SELECT SectionID, SectionNumber
+   FROM getStudentSections($1, $2, $3)
+   WHERE Course ILIKE $4
+   ORDER BY SectionNumber;
+
+$$ LANGUAGE sql
+   STABLE
+   RETURNS NULL ON NULL INPUT;
+
+
+ALTER FUNCTION getStudentSections(studentID INT,
+                                          year NUMERIC(4,0),
+                                          seasonOrder NUMERIC(1,0),
+                                          courseNumber VARCHAR(8)
+                                          )
+   OWNER TO alpha;
+
+REVOKE ALL ON FUNCTION getStudentSections(studentID INT,
+                                          year NUMERIC(4,0),
+                                          seasonOrder NUMERIC(1,0),
+                                          courseNumber VARCHAR(8)
+                                          )
+   FROM PUBLIC;
+
+GRANT EXECUTE ON FUNCTION getStudentSections(studentID INT,
+                                          year NUMERIC(4,0),
```

Comment:
```suggestion
                                             year NUMERIC(4,0),
```

In general, all parameters should line up with the first parameter (like on lines 543-546). I would repeat this with the other parameter lists added.

Reply:
Good catch, thanks!